packages/cli/cloud/src/bin.ts

```diff
@@ -0,0 +1,54 @@
+import { Command } from 'commander';
+import { createLogger } from './services/logger';
+import { CLIContext } from './types';
+import { cli } from './index';
+
+function buildStrapiCloudCommand(argv = process.argv, command = new Command()) {
+  const cloudCommands = {
+    loginCommand: cli.login.command,
+    logoutCommand: cli.logout.command,
+    deployCommand: cli.deployProject.command,
+  } as const;
+
+  // Initial program setup
+  command.storeOptionsAsProperties(false).allowUnknownOption(true);
+
+  // Help command
+  command.helpOption('-h, --help', 'Display help for command');
+  command.addHelpCommand('help [command]', 'Display help for command');
+
+  // Debug and silent options
+  command.option('-d, --debug', 'Output extra debugging');
+  command.option('-s, --silent', 'Output less information');
```

Comment:
Shouldn't you check if you have a provided command or not ? I would only add those options when you create the command completely and just add the sub commands when an existing command is provided.

Reply:
Also for debug & silent in particular we might not want them to be added to the entire CLI. we won't know how to deal with the option in most commands but the CLI help will mention them

Reply:
> Shouldn't you check if you have a provided command or not ? I would only add those options when you create the command completely and just add the sub commands when an existing command is provided.

I don't get what you mean here.
Do you mean that we should dynamically check if there is a `--debug` or `--silent` in `process.argv` ?
I have added these two options as it is implemented in the logger but it wasn't part of the possible options.
Also, if I remember correctly my tests, I was unable to get if there was not a --debug option in the command line if commander doesn't know about it. 🤔